### PR TITLE
GG-24882 First letter in first word of error cause converted to upper case.

### PR DESCRIPTION
**Pull request by vsisko**

### CODE REVIEW

File: modules/web-console/frontend/app/services/ErrorParser.service.ts [hunk 1]
```diff
@@ -70,7 +70,8 @@ export default class {
                     const msgStart = err.message.indexOf(': ', causeIdx + CAUSE_STR.length) + 2;
                     const causeEndLine = err.message.indexOf('\n', msgStart);
                     const msgEnd = err.message.indexOf('[', msgStart);
-                    const cause = err.message.substring(msgStart, msgEnd >= 0 && msgEnd < causeEndLine ? msgEnd : causeEndLine);
+                    const cause = err.message.charAt(msgStart).toUpperCase() +
```
Comment (akuznetsov-gridgain) [inline modules/web-console/frontend/app/services/ErrorParser.service.ts]: Lets use lodash here: `_.upperFirst()`

Comment (Klaster1) [reply modules/web-console/frontend/app/services/ErrorParser.service.ts]: I agree with @akuznetsov-gridgain

File: modules/web-console/frontend/app/services/ErrorParser.service.ts [hunk 2]
```diff
@@ -70,8 +70,7 @@ export default class {
                     const msgStart = err.message.indexOf(': ', causeIdx + CAUSE_STR.length) + 2;
                     const causeEndLine = err.message.indexOf('\n', msgStart);
                     const msgEnd = err.message.indexOf('[', msgStart);
-                    const cause = err.message.charAt(msgStart).toUpperCase() +
-                        err.message.substring(msgStart + 1, msgEnd >= 0 && msgEnd < causeEndLine ? msgEnd : causeEndLine);
+                    const cause = _.upperFirst(err.message.substring(msgStart, msgEnd >= 0 && msgEnd < causeEndLine ? msgEnd : causeEndLine));
```
Comment (Klaster1) [inline modules/web-console/frontend/app/services/ErrorParser.service.ts]: Please either import `_` or `upperFirst`. Do not use global lodash.

File: modules/web-console/frontend/app/services/ErrorParser.spec.ts [hunk 3]
```diff
@@ -20,13 +20,31 @@ import JavaTypes from './JavaTypes.service';
 const parser = new ErrorParser(new JavaTypes());
 
 import { assert } from 'chai';
+import foreach from 'lodash/forEach'
```
Comment (Klaster1) [inline modules/web-console/frontend/app/services/ErrorParser.spec.ts]: Why not use `Array#forEach` instead? I have nothing against Lodash, but this looks like an overkill to me.

**Pull request merged**